fvm/transaction.go

```diff
@@ -48,6 +50,18 @@ func (proc *TransactionProcedure) SetTraceSpan(traceSpan otelTrace.Span) {
 	proc.TraceSpan = traceSpan
 }
 
+func (proc *TransactionProcedure) IsSampled() bool {
+	return proc.TraceSpan != nil
+}
+
+func (proc *TransactionProcedure) StartSpanFromProcTraceSpan(
+	tracer module.Tracer, spanName trace.SpanName) otelTrace.Span {
```

Comment:
add extra newlines to clarify the method signature

Reply:
addressed